lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
I can buy configurable having to be set, that makes sense to me.  I don't know that I buy having `writable: true` is needed.  Maybe I am not understanding this right (_entirely possible_), but `writable` would allow the getter property to be assigned to, and that shouldn't matter if `configurable` is true and the getter is deleted first (which here, it is, and allowable).